custom/dfci_swasth/rules/custom_actions.py

```diff
@@ -0,0 +1,81 @@
+from corehq.apps.data_interfaces.models import CaseRuleActionResult
+from corehq.apps.hqcase.utils import update_case, AUTO_UPDATE_XMLNS
+from corehq.form_processor.exceptions import CaseNotFound
+from corehq.form_processor.models import CommCareCase
+from custom.dfci_swasth.constants import (
+    CASE_TYPE_PATIENT,
+    PROP_CCUSER_CASELOAD_CASE_ID,
+    PROP_COUNSELLOR_CLOSED_CASE_LOAD,
+    PROP_COUNSELLOR_LOAD,
+)
+
+
+def update_counsellor_load(patient_case, rule):
+    num_related_updates = 0
+
+    if patient_case.type != CASE_TYPE_PATIENT:
+        return CaseRuleActionResult(num_related_updates=num_related_updates)
```

Comment:
Since  the `num_related_updates` kwarg is zero by default
```suggestion
        return CaseRuleActionResult()
```